app/services/encryption/multi_region_kms_migration/profile_migrator.rb

```diff
@@ -10,21 +10,23 @@ def initialize(profile)
       end
 
       def migrate!
-        if profile.encrypted_pii.blank? || profile.encrypted_pii_recovery.blank?
-          raise "Profile##{profile.id} is missing encrypted_pii or encrypted_pii_recovery"
-        end
+        profile.with_lock do
+          if profile.encrypted_pii.blank? || profile.encrypted_pii_recovery.blank?
+            raise "Profile##{profile.id} is missing encrypted_pii or encrypted_pii_recovery"
+          end
 
-        return if profile.encrypted_pii_multi_region.present? ||
+          next if profile.encrypted_pii_multi_region.present? ||
```

Comment:
`next` and `return` are equivalent in this context I think; is this just for clarity, or is there something deeper going on here?